pkg/kubelet/allocation/allocation_manager.go

```diff
@@ -147,9 +144,9 @@ func NewManager(checkpointDirectory string,
 		containerManager: containerManager,
 		statusManager:    statusManager,
 		admitHandlers:    lifecycle.PodAdmitHandlers{},
-		podcache:         podCache,
 		sourcesReady:     sourcesReady,
 
+		ticker:         time.NewTicker(30 * time.Second),
```

Comment:
nit: define a constant for this and the 3 minute interval.

Reply:
done